### ISSUE TITLE
MCAN MiAS Case Study - Content

### ISSUE THREAD

**Issue by allisonraeck** [virtualredhead/red.portfolio#37]
**Michigan College Access Network**

Campaign development, marketing strategy

A student-facing campaign to promote the Michigan Achievement Scholarship and shift perspectives regarding the attainability and impact of a college education. 

**Not like other scholarships**
In early 2023, Michigan College Access Network (MCAN) led a statewide coalition to promote the Michigan Achievement Scholarship, a new source of financial aid for people graduating high school, not based on merit and intended to expand eligibility for nearly all students. MCAN sought a large-scale, statewide public awareness marketing campaign to help students and their families understand the importance of the scholarship, generate interest, and more broadly, encourage them to pursue college. 

**Meeting students where they are**
In order to reach students from populations that need to hear this message the most, we knew we’d need to meet students where they were. We developed a comprehensive list of communications strategies and mediums to most effectively reach target students and their parents in all 83 counties across the state. This amounted to more than 15 mediums, with different design and messaging variations for each: A campaign microsite, [getmimoney.org](https://getmimoney.org/). In-school materials like posters, letters, and printed mailers. A statewide mass media campaign, with billboards, transit ads, paid social ads on Facebook and Snapchat, digital ads, and ads on video streaming and Spotify. Grassroots marketing materials, such as organic social posts and an op-ed. It was quite the list. 

**Equity through influence**
When developing the campaign theme and supporting content, we aimed to address the foundational challenge: students’ hesitations about attending college, particularly among low-income students of color. Materials emphasized the return on investment of college, and design followed a bold and playful tone. Through all of these, we incorporated Spanish and Arabic translation to remain inclusive and better reach parents and guardians. In the end, this campaign met audiences where they were in an effort to raise awareness, and inspired students to expand their self-aspirations.

**Comment by allisonraeck** [comment]
Photos are here: https://drive.google.com/drive/folders/16UbPCAB79fnzRygOcMJeiie3ozFFqRtP

**Comment by RosieSchrag** [comment]
@tonybredhead updated photos here 
[Archive.zip](https://github.com/virtualredhead/red.portfolio/files/12018547/Archive.zip)

**Comment by allisonraeck** [comment]
@tonybredhead  New content for section 2:

To start, we developed a comprehensive list of communications strategies and mediums that would most effectively reach target students and their parents in all 83 counties across the state. This amounted to more than 15 mediums, with different design and messaging variations for each: A campaign microsite, [getmimoney.org](https://getmimoney.org/). In-school materials like posters, letters, and printed mailers. A statewide mass media campaign, with billboards, transit ads, paid social ads on Facebook and Snapchat, digital ads, and ads on video streaming and Spotify. Grassroots marketing materials, such as organic social posts and an op-ed. It was quite the list.

**Comment by allisonraeck** [comment]
@tonybredhead Hyper-shortened content for the temporary fix - will this work? 

**Not like other scholarships**
In early 2023, Michigan College Access Network (MCAN) led a statewide coalition to promote the Michigan Achievement Scholarship, a new source of financial aid for people graduating high school. MCAN sought a large-scale, statewide public awareness marketing campaign to spread the word and more broadly, encourage people to pursue college.

**Meeting students where they are**
This campaign incorporated more than 15 mediums: A microsite, [getmimoney.org](https://getmimoney.org/). Posters and printed mailers. Billboards, transit ads, paid social ads on Facebook and Snapchat, digital ads, and video streaming and Spotify ads. Organic social posts and an op-ed. It was quite the list.

**Equity through influence**
Through a playful campaign theme, we aimed to address students’ hesitations about attending college, particularly among low-income students of color. To better reach parents and guardians in our target audience, we incorporated Spanish and Arabic translation. In the end, this campaign met audiences where they were in an effort to raise awareness, and inspired students to expand their self-aspirations.
